grails-app/i18n/messages.properties

```diff
@@ -3232,6 +3234,12 @@ react.default.select.typeToSearch.label=Type to search
 react.default.noOptions.label=No options
 react.default.dateInput.placeholder.label=Select a date
 react.default.form.field.required.label=${0} is required
+
+# Date formats
+react.default.defaultDate.format=MM/DD/yyyy hh:mm:ss a z
+react.default.customDate.format=MM/DD/yyyy
+react.default.expiryDate.format=MM/yyyy
```

Comment:
we're only changing the format to dd/MMMMM/yyyy for ex_MX? Do we not want all locals to have the new format?

Reply:
I think that others should have a default right now.